src/TestUtils/src/UITest.Appium/HelperExtensions.cs

```diff
@@ -174,6 +174,24 @@ public static bool WaitForTextToBePresentInElement(this IApp app, string automat
 			}
 		}
 
+		/// <summary>
+		/// Changes the device orientation to landscape mode.
+		/// </summary>
+		/// <param name="app">Represents the main gateway to interact with an app.</param>
+		public static void SetOrientationLandscape(this IApp app)
```

Comment:
Same naming we had in Xamarin.UITest to allow easy porting of tests https://learn.microsoft.com/en-us/dotnet/api/xamarin.uitest.iapp.setorientationlandscape?view=xamarin-uitest-sdk